js/flightlog_parser.js

```diff
@@ -1511,6 +1550,7 @@ var FlightLogParser = function(logData) {
         }
 
         adjustFieldDefsList(that.sysConfig.firmwareType, that.sysConfig.firmwareVersion);
+        FlightLogFieldPresenter.adjustDebugDefsList(that.sysConfig.firmwareType, that.sysConfig.firmwareVersion);
```

Comment:
This is from other PR... this don't need to be here. A rebase of your code will include it by default, it was merged.

Reply:
Yes I'll fix that